src/Service/PaymentRestrictionValidation.php

```diff
@@ -28,11 +28,11 @@
 class PaymentRestrictionValidation
 {
     /**
-     * @var \Traversable
+     * @var array
```

Comment:
add typehint of what kind of array it its. Now the array doesn't tell me anything about what information is given